docs/pbc/all/ratings-reviews/202212.0/third-party-integrations/integrate-bazaarvoice.md

```diff
@@ -266,5 +217,79 @@ $jobs[] = [
     'stores' => $allStores,
 ];
 ```
+
+### 4. Configure OMS
+
+#### Extend command plugins
+
+Add the following plugin to `src/Pyz/Zed/Oms/OmsDependencyProvider.php`:
+
+```php
+use Spryker\Zed\Oms\Communication\Plugin\Oms\Command\SendOrderStatusChangedMessagePlugin;
+
+// ...
+
+/**
+ * @param \Spryker\Zed\Kernel\Container $container
+ *
+ * @return \Spryker\Zed\Kernel\Container
+ */
+protected function extendCommandPlugins(Container $container): Container
+{
+    $container->extend(self::COMMAND_PLUGINS, function (CommandCollectionInterface $commandCollection) {
+        // ...
+        $commandCollection->add(new SendOrderStatusChangedMessagePlugin(), 'Order/RequestProductReviews');
+    
+        return $commandCollection;
+    });
+ }
+```
+
+#### Add order hydration plugin (Marketplace only)
+
+Add the following plugin to `src/Pyz/Zed/Sales/SalesDependencyProvider.php`:
+
+```php
+use Spryker\Zed\MerchantProfile\Communication\Plugin\Sales\MerchantDataOrderHydratePlugin;
+
+// ...
+
+/**
+ * @return array<\Spryker\Zed\SalesExtension\Dependency\Plugin\OrderExpanderPluginInterface>
+ */
+protected function getOrderHydrationPlugins(): array
+{
+    return [
+        // ...
+        new MerchantDataOrderHydratePlugin(),
+    ];
+}
+```
+
+#### Update OMS schema (Marketplace only)
```

Comment:
Why is it only for marketplace? This is normally something that all projects must do.